test/fixtures/config/config-with-exclude-labels.yml

```diff
@@ -4,8 +4,10 @@ template: |
   $CHANGES
 
 categories:
+  - label: feature
+    title: 🚀 Features
   - label: fix
     title: 🐛 Bug Fixes
 
 exclude-labels:
-  - feature
+  - skip-changelog
```

Comment:
👍

Reply:
perhaps update the example here: https://github.com/toolmantim/release-drafter/pull/164/commits/a6c6b0100ce06bc22faab8d3edd066917a94dbfa

Reply:
Yup, good idea!